README.md

```diff
@@ -0,0 +1,128 @@
+# CCCD Migrator
+
+Ruby cli that wraps calls to aws cli commandline utility for s3 synchronization
+between template deploy environment's s3 buckets (source) and live-1 s3 buckets (destination).
+
+## Build
+To build the docker image using the provided script you will need credentials for accessing live-1 cccd ECR registry. Then you can
```

Comment:
Would possibly add a little more help here for people less familar with aws? - I think the user would have to set up a set of credentials specifically named ecr-live1.

Reply:
cool, will do. although, if they don't know that much about it they should probably not do it 😬  - but will mention the assumed profile-name